### PR TITLE
CHANGELOG for 1.13.0

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+ * Provide syntax coloration for #fragment. See [#570](https://github.com/redhat-developer/vscode-quarkus/pull/570).
+
+### Bug Fixes
+
+ * Treat `CompletionStage` and `Uni` objects as their resolved type for validation and completion in Qute templates. See [redhat-developer/quarkus-ls#826](https://github.com/redhat-developer/quarkus-ls/issues/826).
+ * `{#if}` sections in Qute templates of the form `{#if !inject:beanName}{/if}` are marked as invalid. See [redhat-developer/quarkus-ls#828](https://github.com/redhat-developer/quarkus-ls/issues/828).
+ * Quick fix to generate a `@TemplateExtension` class from a Qute template might place the class in the wrong folder. See [redhat-developer/quarkus-ls#831](https://github.com/redhat-developer/quarkus-ls/issues/831).
+ * "Create project" fails when there are no folders and one java file open. See [#559](https://github.com/redhat-developer/vscode-quarkus/issues/559).
+ * Fix Quarkus debug configuration after maven wrapper is deleted. See [#572](https://github.com/redhat-developer/vscode-quarkus/issues/572).
+ * qute-ls isn't stopped by vscode-quarkus. See [#576](https://github.com/redhat-developer/vscode-quarkus/issues/576).
```
Comment (JessicaJHee) [inline CHANGELOG.md]: Maybe rename to something like:
`qute-ls remains running after vscode-quarkus is stopped`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+ * Prevent infinite loop when going to the definition of a user tag. See [#579](https://github.com/redhat-developer/vscode-quarkus/issues/579).
+ * Prevent popups by reading files without triggering `onDidOpenTextDocument`. See [#584](https://github.com/redhat-developer/vscode-quarkus/pull/584).
+ * Prevent errors while vscode-java is still starting. See [#586](https://github.com/redhat-developer/vscode-quarkus/issues/586).
+ * Incomplete `{#if}` section with operator may generate `ClassCastException`. See [redhat-developer/quarkus-ls#816](https://github.com/redhat-developer/quarkus-ls/issues/816).
+ * Use quarkus-ls's Qute parser for syntax validation in order to get consistent diagnostic positions and ranges. See [redhat-developer/quarkus-ls#812](https://github.com/redhat-developer/quarkus-ls/issues/812).
+ * Completion in Qute template should generate only start section when an empty end section is found. See [redhat-developer/quarkus-ls#805](https://github.com/redhat-developer/quarkus-ls/issues/805).
+ * Quarkus 3 support. See [redhat-developer/quarkus-ls#779](https://github.com/redhat-developer/quarkus-ls/issues/779).
```
Comment (JessicaJHee) [inline CHANGELOG.md]: Would this be considered a bug fix?

**Review by JessicaJHee — COMMENTED** [review]
A few more issues that we may want to mention:
* https://github.com/redhat-developer/vscode-quarkus/issues/587
* https://github.com/redhat-developer/vscode-quarkus/issues/569
* https://github.com/redhat-developer/vscode-quarkus/issues/505

File: CHANGELOG.md (hunk 2 above)
Comment (datho7561) [reply CHANGELOG.md]: I listed it as a bug under quarkus-ls, but I agree it's an important feature for this release, so I'll move it to features here.

**Comment by datho7561** [comment]
I didn't include #587, since the change to add the `uri`/`uriabs` support is listed on the changelog, and I think it would be a bit redundant to include both the issue for the feature and the issue for the documentation.

**Pull request merged**